src/defs/policyRepresentation.ts

```diff
@@ -8,6 +8,11 @@ export enum DecisionStrategy {
   CONSENSUS = 'CONSENSUS',
 }
 
+export enum DecisionEffect {
+  PERMIT = 'PERMIT',
+  DENY = 'DENY',
```

Comment:
Let's use the same conventions here as in the [TypeScript handbook](https://www.typescriptlang.org/docs/handbook/enums.html), which is to camel-case the name of the key.
```suggestion
  Permit = 'PERMIT',
  Deny = 'DENY',
```